### PR TITLE
Clarify how to check for modifier keys in Keyboard events

### PR DESCRIPTION
**Pull request by citelao**
The documentation surrounding keyboard events did not match the examples given, since there are actually two proposed methods of getting modifier key state.

This change documents both methods in more detail.

### CODE REVIEW

**Comment by PRMerger9** [comment]
@citelao : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-apps-src/design/input/keyboard-events.md [hunk 1]
```diff
@@ -103,13 +103,15 @@ The [**KeyDown**](https://docs.microsoft.com/uwp/api/windows.ui.xaml.uielement.k
...
+Alternatively, you can use the [**GetKeyState()**](https://docs.microsoft.com/uwp/api/windows.ui.core.corewindow.getkeystate) function of the [**CoreWindow**](https://docs.microsoft.com/uwp/api/windows.ui.core.corewindow) (available via [**CoreWindow::GetForCurrentThread()**](https://docs.microsoft.comuwp/api/windows.ui.core.corewindow.getforcurrentthread), for example) to check modifier states on-demand when a non-modifier key is pressed.
+
+The following examples implement this second method while also including stub code for the first implementation.
 
 > [!NOTE]
 > The Alt key is represented by the **VirtualKey.Menu** value.
 
- 
-
 ### Shortcut keys example
```
Comment (citelao) [inline windows-apps-src/design/input/keyboard-events.md:116]: The examples given don't fully implement the first strategy.

They include `KeyDown` handlers to store when ctrl is first pressed, but they don't ever update that value when the key is up:

```c++
void KeyboardSupport::MainPage::Grid_KeyDown(Platform::Object^ sender, Windows::UI::Xaml::Input::KeyRoutedEventArgs^ e)
{
    if (e->Key == VirtualKey::Control) isCtrlKeyPressed = true;
}


void KeyboardSupport::MainPage::Grid_KeyUp(Platform::Object^ sender, Windows::UI::Xaml::Input::KeyRoutedEventArgs^ e)
{
    if (IsCtrlKeyPressed()) 
    {
        if (e->Key==VirtualKey::P) { DemoMovie->Play(); }
        if (e->Key==VirtualKey::A) { DemoMovie->Pause(); }
        if (e->Key==VirtualKey::S) { DemoMovie->Stop(); }
    }
}
```

The C# example does not have this extra code (though I think the samples should include both methods and explain them).

**Comment by citelao** [comment]
@Karl-Bridge-Microsoft: I've proposed a small docs change here, but I think we need to update the code samples as well.

**Comment by ktoliver** [comment]
@citelao @Karl-Bridge-Microsoft - If the update is approved, would you correct the URL in line 108 to add the missing **/** (.com/uwp)? Thanks.

**Comment by PRMerger15** [comment]
@citelao : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by citelao** [comment]
> @citelao @Karl-Bridge-Microsoft - If the update is approved, would you correct the URL in line 108 to add the missing **/** (.com/uwp)? Thanks.

Fixed :D

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
I resolved the merge conflict. There's an update in master, added since this PR was opened, that changes line 106. So, it's incorporated into this PR.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Comment by PRMerger7** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**